docs/upstream-features.md

```diff
@@ -2,37 +2,45 @@
 
 Some ESI features will require enhancement to upstream OpenStack projects.
 
-## Ironic: Multi-tenant support
+## Current Work
 
-ESI requires multi-tenant support in Ironic to meet our basic functionality. This enables a tenant to “own” a baremetal node — that is, to have exclusive access to it — without requiring administrative privileges.
+### Ironic: Attestation support with Keylime
 
-An initial implementation of this feature has merged as [If08586f3e9705dd38ff83e4b500d9ee3cd45bce3][]. There is a detailed description of the change available in the [associated blueprint][I1e898f7b9791aa579a733996711945e273ef6a4a].
+[Keylime][] provides attestation — that is, assurance that a given baremetal node has not been tampered with. We would like to integrate this into the Ironic provisioning workflow such that prior to provisioning a node Ironic will run it through an attestation step, and will fail the node if the attestation fails.
```

Comment:
For ESI, do we only need to consider an attestation during node cleaning when it's returned from the lessee?

Reply:
I'd say the answer is up in the air :) I didn't actually change this text; just moved it around.